astrodata/testing.py

```diff
@@ -371,6 +399,10 @@ def path_to_inputs(request, env_var='DRAGONS_TEST'):
         pytest.fail('\n  Path to input test data exists but is not accessible: '
                     '\n    {:s}'.format(path))
 
+    branch_name = get_active_git_branch()
+    path_with_branch = os.path.join(path, branch_name)
+    path = path_with_branch if os.path.exists(path_with_branch) else path
```

Comment:
@saimn Could you clarify or give an example of what do you mean with "it would be easier if the branch-specific directories were at the same level"? I am not sure I understood correctly.

Reply:
With the current code the directory for a given branch would be `some_test/refs/branch_name` right ? I was thinking that it may be easier to have something like `some_test/refs_<branch_name>`. This way when the branch is merged you just have to delete the old `refs/` and rename the other directory.

Reply:
Also, and related to the other comment above, if `branch_name` returned by `get_active_git_branch` is an empty string, `os.path.join(path, branch_name)` would work and use the default directory, but it's not the intent of the code.

Reply:
I agree with your first comment. I had a concern that a branch named `feature/something` would not be properly parsed, but I believe it should be Ok.

Regarding your second, I think I understood what you mean and I will try to apply it.

Reply:
Oh indeed, branch names with a `/` could be annoying... Maybe do a `.replace('/', '-')` on the branch name ?